scripts/profile-compiler-build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# To be run from the root of the Flambda backend repo
+
+set -e -u -o pipefail
+
+dump_dir="`pwd`/profile"
+summary_path="`pwd`/summary.csv"
+
+export OLD_OCAMLPARAM="${OCAMLPARAM:-}"
+export OCAMLPARAM="_,profile=1,dump-into-csv=1,dump-dir=$dump_dir,regalloc=irc,regalloc-param=SPLIT_LIVE_RANGES:on,regalloc-param=IRC_SPILLING_HEURISTICS:flat-uses"
```

Comment:
```suggestion
export OCAMLPARAM="_,profile=1,dump-into-csv=1,dump-dir=$dump_dir,regalloc=irc"
```
(simplification possible after https://github.com/ocaml-flambda/flambda-backend/pull/2855)